README.adoc

```diff
@@ -15,6 +15,8 @@
 :page-tags: ['MicroProfile']
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['rest-intro', 'cdi-intro', 'microprofile-config', 'circuit-breaker']
+:page-seo-title: Building fault-tolerance microservices with the MicroProfile Fault Tolerance @Fallback annotation
```

Comment:
Change to:
"Build fault-tolerant microservices with the `@Fallback` MicroProfile Fault Tolerance annotation."

Reply:
Kept the original as the title.

Reply:
I think we mentioned updating this to:
`Building fault-tolerant microservices with the MicroProfile Fault Tolerance @Fallback annotation`

I think fault-tolerant works better here as an adjective, like in https://developer.ibm.com/patterns/build-fault-tolerant-microservices-with-java/

Reply:
That's the same as in the original text.

Reply:
Thank you for pointing out the `fault-tolerant` word, it's fixed in https://github.com/OpenLiberty/guide-microprofile-fallback/pull/101/commits/526df5bbb147ce7db78fb49b9d4d769a4ccea5c8.